advisories/hackage/biscuit-haskell/HSEC-0000-0000.md

```diff
@@ -0,0 +1,45 @@
+```toml
+[advisory]
+# Identifier for the advisory (mandatory). Will be assigned a "HSEC-YYYY-NNNN"
+# identifier e.g. HSEC-2022-0001. Please use "HSEC-0000-0000" in PRs.
+id = "HSEC-0000-0000"
```

Comment:
Please use HSEC-2024-0009 for this advisory.

Reply:
done